content/blog/5-things-you-can-do-with-survey-forms.md

```diff
@@ -6,6 +6,7 @@ author: Mriganka Chandra
 createdAt: 2022-12-08T05:38:26.147Z
 coverImg: /img/five_things_you_can_do_with_survey__hero.png
 coverImgAlt: Five things you can do with a survey form
+images: https://formester.com/img/five_things_you_can_do_with_survey__hero.png
```

Comment:
As it is `images` it should always be an array even if there are only one item

Reply:
Yes, Fixed @ankitsinghaniyaz